ios/TextileNode.m

```diff
@@ -384,12 +418,12 @@ - (dispatch_queue_t)methodQueue {
   [self fulfillWithResult:nil error:error resolver:resolve rejecter:reject];
 }
 
-RCT_EXPORT_METHOD(newTextile:(NSString*)repoPath logLevels:(NSString*)logLevels resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+RCT_EXPORT_METHOD(newTextile:(NSString*)repoPath debug:(BOOL)debug resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
```

Comment:
Seeing there are two places where `debug` is set, here and above in the init config. Is there a difference? Do we need both?

Reply:
yeah, both are needed, or rather, both are useful... init generates debug information as well as newTextile